### ISSUE TITLE
still trying to get tests going on windows.   

### ISSUE THREAD

**Issue by sbillinge** [regro/regolith#248]
Xonsh can't find regolith?????

```
simon@LAPTOP-FU6OME5O MINGW64 ~/dev/regolith (fix_path_in_tests)
$ python run_tests.py
pytest arguments: ['-v']
============================= test session starts =============================
platform win32 -- Python 3.6.6, pytest-3.6.4, py-1.5.2, pluggy-0.6.0 -- C:\Users\simon\Anaconda3\envs\bg\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\simon\dev\regolith, inifile:
plugins: xonsh-0.7.7
collecting ... collected 80 items

tests/test_broker.py::test_round_trip PASSED                             [  1%]
tests/test_builders.py::test_builder[cv] FAILED                          [  2%]
tests/test_builders.py::test_builder[html] FAILED                        [  3%]
tests/test_builders.py::test_builder[resume] FAILED                      [  5%]
tests/test_builders.py::test_builder[publist] FAILED                     [  6%]
tests/test_builders.py::test_builder[current-pending] FAILED             [  7%]
tests/test_builders.py::test_builder[preslist] FAILED                    [  8%]
tests/test_builders.py::test_builder[reimb] FAILED                       [ 10%]
tests/test_builders.py::test_builder[figure] FAILED                      [ 11%]
tests/test_builders.py::test_builder_python[cv] PASSED                   [ 12%]
tests/test_builders.py::test_builder_python[html] FAILED                 [ 13%]
tests/test_builders.py::test_builder_python[resume] PASSED               [ 15%]
tests/test_builders.py::test_builder_python[publist] PASSED              [ 16%]
tests/test_builders.py::test_builder_python[current-pending] PASSED      [ 17%]
tests/test_builders.py::test_builder_python[preslist] PASSED             [ 18%]
tests/test_builders.py::test_builder_python[reimb] PASSED                [ 20%]
tests/test_builders.py::test_builder_python[figure] FAILED               [ 21%]
tests/test_chain_db.py::test_dddi PASSED                                 [ 22%]
tests/test_chain_db.py::test_second_mapping PASSED                       [ 23%]
tests/test_chain_db.py::test_double_mapping PASSED                       [ 25%]
tests/test_chain_db.py::test_list_mapping PASSED                         [ 26%]
tests/test_chain_db.py::test_mixed_mapping PASSED                        [ 27%]
tests/test_chain_db.py::test_exactness PASSED                            [ 28%]
tests/test_chain_db.py::test_exactness_setting PASSED                    [ 30%]
tests/test_chain_db.py::test_exactness_setting_multi PASSED              [ 31%]
tests/test_chain_db.py::test_exactness_setting_multi2 PASSED             [ 32%]
tests/test_chain_db.py::test_exactness_setting_multi_novel PASSED        [ 33%]
tests/test_chain_db.py::test_dicts_in_lists PASSED                       [ 35%]
tests/test_chain_db.py::test_dicts_in_lists_mutation PASSED              [ 36%]
tests/test_main.py::test_user_rc PASSED                                  [ 37%]
tests/test_storage.py::test_cmd FAILED                                   [ 38%]
tests/test_tools.py::test_author_publications PASSED                     [ 40%]
tests/test_tools.py::test_fuzzy_retrieval PASSED                         [ 41%]
tests/test_tools.py::test_number_suffix[0-th] PASSED                     [ 42%]
tests/test_tools.py::test_number_suffix[1-st] PASSED                     [ 43%]
tests/test_tools.py::test_number_suffix[2-nd] PASSED                     [ 45%]
tests/test_tools.py::test_number_suffix[3-rd] PASSED                     [ 46%]
tests/test_tools.py::test_number_suffix[4-th] PASSED                     [ 47%]
tests/test_tools.py::test_number_suffix[10-th] PASSED                    [ 48%]
tests/test_tools.py::test_number_suffix[13-th] PASSED                    [ 50%]
tests/test_tools.py::test_number_suffix[33-rd] PASSED                    [ 51%]
tests/test_tools.py::test_number_suffix[None-] PASSED                    [ 52%]
tests/test_tools.py::test_number_suffix[0-] PASSED                       [ 53%]
tests/test_tools.py::test_latex_safe[$hi-\\$hi-kwargs0] PASSED           [ 55%]
tests/test_tools.py::test_latex_safe[Website: https://github.com/CJ-Wright/Masters_Thesis/raw/master/thesis.pdf hi-Website: \\url{https://github.com/CJ-Wright/Masters_Thesis/raw/master/thesis.pdf} hi-kwargs1] PASSED [ 56%]
tests/test_tools.py::test_latex_safe[Website: https://github.com/CJ-Wright/Masters_Thesis/raw/master/thesis.pdf hi-Website: \\href{https://github.com/CJ-Wright/Masters_Thesis/raw/master/thesis.pdf} hi-kwargs2] PASSED [ 57%]
tests/test_tools.py::test_latex_safe[Website: https://github.com/CJ-Wright/Masters_Thesis/raw/master/thesis.pdf hi-Website: https://github.com/CJ-Wright/Masters\\_Thesis/raw/master/thesis.pdf hi-kwargs3] PASSED [ 58%]
tests/test_validate.py::test_validate_python PASSED                      [ 60%]
tests/test_validate.py::test_validate_python_single_col PASSED           [ 61%]
tests/test_validate.py::test_validate_bad_python PASSED                  [ 62%]
tests/test_validate.py::test_validate FAILED                             [ 63%]
tests/test_validate.py::test_validate_bad FAILED                         [ 65%]
tests/test_validators.py::test_validation[abstracts] PASSED              [ 66%]
tests/test_validators.py::test_validation[assignments] PASSED            [ 67%]
tests/test_validators.py::test_validation[blog] PASSED                   [ 68%]
tests/test_validators.py::test_validation[grades] PASSED                 [ 70%]
tests/test_validators.py::test_validation[grants] PASSED                 [ 71%]
tests/test_validators.py::test_validation[groups] PASSED                 [ 72%]
tests/test_validators.py::test_validation[institutions] PASSED           [ 73%]
tests/test_validators.py::test_validation[people] PASSED                 [ 75%]
tests/test_validators.py::test_validation[presentations] PASSED          [ 76%]
tests/test_validators.py::test_validation[projects] PASSED               [ 77%]
tests/test_validators.py::test_validation[proposals] PASSED              [ 78%]
tests/test_validators.py::test_validation[refereeReports] PASSED         [ 80%]
tests/test_validators.py::test_validation[students] PASSED               [ 81%]
tests/test_validators.py::test_validation[expenses] PASSED               [ 82%]
tests/test_validators.py::test_exemplars[abstracts] PASSED               [ 83%]
tests/test_validators.py::test_exemplars[assignments] PASSED             [ 85%]
tests/test_validators.py::test_exemplars[blog] PASSED                    [ 86%]
tests/test_validators.py::test_exemplars[grades] PASSED                  [ 87%]
tests/test_validators.py::test_exemplars[grants] PASSED                  [ 88%]
tests/test_validators.py::test_exemplars[groups] PASSED                  [ 90%]
tests/test_validators.py::test_exemplars[institutions] PASSED            [ 91%]
tests/test_validators.py::test_exemplars[people] PASSED                  [ 92%]
tests/test_validators.py::test_exemplars[presentations] PASSED           [ 93%]
tests/test_validators.py::test_exemplars[projects] PASSED                [ 95%]
tests/test_validators.py::test_exemplars[proposals] PASSED               [ 96%]
tests/test_validators.py::test_exemplars[refereeReports] PASSED          [ 97%]
tests/test_validators.py::test_exemplars[students] PASSED                [ 98%]
tests/test_validators.py::test_exemplars[expenses] PASSED                [100%]

================================== FAILURES ===================================
______________________________ test_builder[cv] _______________________________

bm = 'cv', make_db = 'C:\\Users\\simon\\AppData\\Local\\Temp\\regolith_fake'

    @pytest.mark.parametrize("bm", builder_map)
    def test_builder(bm, make_db):
        repo = make_db
        os.chdir(repo)
        if bm == "figure":
            prep_figure()
        if bm == "html":
            os.makedirs("templates/static", exist_ok=True)
>       subprocess.run(["regolith", "build", bm, "--no-pdf"], check=True, cwd=repo)

tests\test_builders.py:51:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\lib\subprocess.xsh:16: in run
    p = ![@(cmd)]
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:19783: in subproc_captured_hiddenobject
    return run_subproc(cmds, captured='hiddenobject')
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:19746: in run_subproc
    command.end()
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:12899: in end
    self._end(tee_output=tee_output)
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:12918: in _end
    self._raise_subproc_error()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self =

    def _raise_subproc_error(self):
        """Raises a subprocess error, if we are supposed to."""
        spec = self.spec
        rtn = self.returncode
        if (not spec.is_proxy and
                rtn is not None and
                rtn > 0 and
                builtins.__xonsh_env__.get('RAISE_SUBPROC_ERROR')):
            try:
                raise subprocess.CalledProcessError(rtn, spec.cmd,
>                                                   output=self.output)
E                                                   subprocess.CalledProcessError: Command '['regolith', 'build', 'cv', '--no-pdf']' returned non-zero exit status 1.

..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:13039: CalledProcessError
---------------------------- Captured stdout call -----------------------------
~\AppData\Local\Temp\regolith_fake ~\dev\regolith
~\dev\regolith
---------------------------- Captured stderr call -----------------------------
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
xonsh: subprocess mode: command not found: regolith
_____________________________ test_builder[html] ______________________________

bm = 'html', make_db = 'C:\\Users\\simon\\AppData\\Local\\Temp\\regolith_fake'

    @pytest.mark.parametrize("bm", builder_map)
    def test_builder(bm, make_db):
        repo = make_db
        os.chdir(repo)
        if bm == "figure":
            prep_figure()
        if bm == "html":
            os.makedirs("templates/static", exist_ok=True)
>       subprocess.run(["regolith", "build", bm, "--no-pdf"], check=True, cwd=repo)

tests\test_builders.py:51:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\lib\subprocess.xsh:16: in run
    p = ![@(cmd)]
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:19783: in subproc_captured_hiddenobject
    return run_subproc(cmds, captured='hiddenobject')
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:19746: in run_subproc
    command.end()
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:12899: in end
    self._end(tee_output=tee_output)
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:12918: in _end
    self._raise_subproc_error()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self =

    def _raise_subproc_error(self):
        """Raises a subprocess error, if we are supposed to."""
        spec = self.spec
        rtn = self.returncode
        if (not spec.is_proxy and
                rtn is not None and
                rtn > 0 and
                builtins.__xonsh_env__.get('RAISE_SUBPROC_ERROR')):
            try:
                raise subprocess.CalledProcessError(rtn, spec.cmd,
>                                                   output=self.output)
E                                                   subprocess.CalledProcessError: Command '['regolith', 'build', 'html', '--no-pdf']' returned non-zero exit status 1.

..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:13039: CalledProcessError
---------------------------- Captured stdout call -----------------------------
~\AppData\Local\Temp\regolith_fake ~\dev\regolith
~\dev\regolith
---------------------------- Captured stderr call -----------------------------
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
xonsh: subprocess mode: command not found: regolith
____________________________ test_builder[resume] _____________________________

bm = 'resume', make_db = 'C:\\Users\\simon\\AppData\\Local\\Temp\\regolith_fake'

    @pytest.mark.parametrize("bm", builder_map)
    def test_builder(bm, make_db):
        repo = make_db
        os.chdir(repo)
        if bm == "figure":
            prep_figure()
        if bm == "html":
            os.makedirs("templates/static", exist_ok=True)
>       subprocess.run(["regolith", "build", bm, "--no-pdf"], check=True, cwd=repo)

tests\test_builders.py:51:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\lib\subprocess.xsh:16: in run
    p = ![@(cmd)]
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:19783: in subproc_captured_hiddenobject
    return run_subproc(cmds, captured='hiddenobject')
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:19746: in run_subproc
    command.end()
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:12899: in end
    self._end(tee_output=tee_output)
..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:12918: in _end
    self._raise_subproc_error()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self =

    def _raise_subproc_error(self):
        """Raises a subprocess error, if we are supposed to."""
        spec = self.spec
        rtn = self.returncode
        if (not spec.is_proxy and
                rtn is not None and
                rtn > 0 and
                builtins.__xonsh_env__.get('RAISE_SUBPROC_ERROR')):
            try:
                raise subprocess.CalledProcessError(rtn, spec.cmd,
>                                                   output=self.output)
E                                                   subprocess.CalledProcessError: Command '['regolith', 'build', 'resume', '--no-pdf']' returned non-zero exit status 1.

..\..\Anaconda3\envs\bg\lib\site-packages\xonsh\__amalgam__.py:13039: CalledProcessError
---------------------------- Captured stdout call -----------------------------
~\AppData\Local\Temp\regolith_fake ~\dev\regolith
~\dev\regolith
---------------------------- Captured stderr call -----------------------------
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
xonsh: subprocess mode: command not found: regolith
____________________________ test_builder[publist] ____________________________
```

**Comment by sbillinge** [comment]
@scopatz not sure if you get tautomatic notifications.  Don't spend time on this, but if you have any ideas, they are welcome

**Comment by scopatz** [comment]
Weird, it seems like it might be that on windows it needs to be `regolith.exe`

**Comment by sbillinge** [comment]
Closed by #247
